pyproject.toml

```diff
@@ -1,3 +1,102 @@
+[tool.poetry]
+name = "saleor"
+version = "2.8.0"
+description = "A modular, high performance e-commerce storefront built with GraphQL, Django, and ReactJS."
+authors = ["Mirumee Software <hello@mirumee.com>"]
+license = "BSD-3-Clause"
+
+readme = "README.md"
+
+homepage = "https://getsaleor.com/"
+repository = "https://github.com/mirumee/saleor"
+documentation = "https://docs.getsaleor.com/en/latest/"
+
+[tool.poetry.dependencies]
+python = "~3.7"
+babel = "~2.7"
+bleach = "~2"
+boto3 = "^1.9"
+braintree = "~3.49.0"
+celery = { version = "^4.3.0", extras = ["redis"] }
+dataclasses = { version = "^0.6", markers = "python_version < '3.7'" }
+dj-database-url = "^0"
+dj-email-url = "^0"
+django = "~2.2"
+django-babel = "^0"
+django-bootstrap4 = "^0"
+django-cache-url = "^3.0"
+django-countries = "^5.3"
+django-elasticsearch-dsl = "^0.5"
+django-filter = "^2.2"
+django-graphql-jwt = "^0"
```

Comment:
We need to lock it to `<=0.2.1` (c.f. https://github.com/mirumee/saleor/pull/4653 for explanation)

Reply:
Thanks, done 👍